packages/effects-runtime/test/__fixtures__/functional-tests/errors-in-handlers.js

```diff
@@ -38,25 +32,12 @@ const asyncEjectCase = async () => {
     }
 };
 
-const continuationEjectCase = () => {
-  'use effects';
-  try{
-
-  }handle ejectType with (e){
-      continuationEject();
-  }
-};
-
-module.exports.test = async ({it, expect, code}) => {
+module.exports.test = async ({it, expect}) => {
     it('Should handle errors as expected when effect handlers throw', async() => {
         await expect(syncEjectCase()).rejects.toThrowError('eject');
     });
 
     it('Should handle errors as expected when effect handlers are async and throw', async () => {
         await expect(asyncEjectCase()).rejects.toThrowError('eject');
     });
-
-    it.skip('Should handle errors as expected when effect handlers are async and throw', async () => {
```

Comment:
Removing this because if something like

```
setTimeout(() => {
        throw new Error('eject');
    }, 10);
```

actually _did_ throw, that would be the opposite of what was expected.